packages/ia-userlist-settings/test/ia-user-list-settings.test.ts

```diff
@@ -43,15 +43,19 @@ describe('IAUserListSettings', () => {
 
     const newListElement = el.shadowRoot?.querySelector('.new-list');
 
-    const listId = newListElement?.querySelector('#id');
-    const listName = newListElement?.querySelector('#name');
-    const listDesc = newListElement?.querySelector('#description');
-    const listPrivate = newListElement?.querySelector('#private');
+    const listId = newListElement?.querySelector('#id') as HTMLInputElement;
+    const listName = newListElement?.querySelector('#name') as HTMLInputElement;
+    const listDesc = newListElement?.querySelector(
+      '#description'
+    ) as HTMLInputElement;
+    const listPrivate = newListElement?.querySelector(
+      '#private'
+    ) as HTMLInputElement;
 
-    expect(listId?.textContent).to.equal('');
```

Comment:
@nsharma123 
Question: I don't think that `textContent` tests the is the correct property here.